SConstruct

```diff
@@ -12,6 +12,13 @@ import fnmatch
 from SCons.Node.Alias import default_ans
 import time
 
+try:
+    enscons_active = True
+    import pytoml as toml
+    import enscons, enscons.cpyext
+except ImportError:
+    enscons_active = False
+
```

Comment:
This makes sure that even if someone is building it without cibuildwheel+enscons, this script can still run normally.